### ISSUE TITLE
No posh support for ASP.NET 5 apps

### ISSUE THREAD

**Issue by robs** [projectkudu/KuduScript#36]
Hi Guys,

I appreciate you've only just added PowerShell script generation but I was going to try it out on an Asp.Net 5 site.

I'm not sure if you just forgot or if there was a reason it didn't get included in the posh support updates?

I can't see any templates for it so I guess there is a reason.

Going to translate over from the .cmd version but just thought I'd flag it up :)

**Comment by ahmelsayed** [comment]
You're right there is no PowerShell deployment script for AspNet 5. I was going to add one but it wasn't urgent and I thought I'd wait until RC since they are looking at more changes in DNVM and how they want to handle deployments on Azure in general. We will eventually add one and I'll use this issue for tracking that.

**Comment by robs** [comment]
Fair enough, thanks for getting back to me.

I've got it working based on deploy.cmd anyway.

On a side note, do you plan on adding a param for a test project location?
Happy to add an issue for it but it's not really an "issue" so don't want to make it look like a bug.

I bet loads of people would love that support out of the box though.

**Comment by davidebbo** [comment]
@robs best to discuss separately, or we risk losing track of what this issue is tracking (it's quite unrelated).
